uisdk/src/adyen/java/com/karhoo/uisdk/AdyenDropInServicePresenter.kt

```diff
@@ -128,6 +128,7 @@ class AdyenDropInServicePresenter(
         val request = JSONObject()
         request.put(PAYMENTS_PAYLOAD, payload)
         request.put(SUPPLY_PARTNER_ID, getSupplyPartnerId())
+        request.put(CONSENT_MODE_SUPPORTED, true)
```

Comment:
On iOS I pass the value set to reference of `.setShowStorePaymentField` property, not hardcoded `true`. What approach is a correct one?

Reply:
The two values should be always aligned. Whether we show or don't show the toggle influences the backend logic. So always sending true is incorrect and can cause odd behaviour